curriculum/challenges/english/01-responsive-web-design/applied-visual-design/center-an-element-horizontally-using-the-margin-property.md

```diff
@@ -21,9 +21,9 @@ Center the `div` on the page by adding a `margin` property with a value of `auto
 
 The `div` should have a `margin` set to `auto`.
 
-```js
-assert(code.match(/margin:\s*?auto;/g));
-```
+```js 
+assert(new __helpers.CSSHelp(document).getStyle('div')?.margin === 'auto');
+``` 
```

Comment:
````suggestion
```js
assert(new __helpers.CSSHelp(document).getStyle('div')?.margin === 'auto');
```
````

Reply:
Sorry if this sounds misinformed and this is just for my curiosity, but what would be the purpose of removing the trailing white-space?

Reply:
No worries about asking 🙂 

a) To standardise the code in the codebase
b) The Linter might/could/should throw warnings about this sort of thing. So, removing them ensures the Linter does not complain at a later date.

Reply:
Ah thank you :) I've committed this along with my attempted fix for this pr so 🤞this will all be resolved. (I didn't think a simple change could become so messy 💦)